openstax/tutor-js
# Copying tasks from past courses

Needs https://github.com/openstax/tutor-server/pull/1301  - without that it can't load the list of offerings

To test:

 - Create a copy of a course that has assignments using the "New Course" option from menu. 
 - Go into the cloned course and open add assignment menu.
   - Menu should display as a sidebar
   - Past assignments can be dragged from sidebar onto cal

![screen shot 2016-11-01 at 4 27 21 pm](https://cloud.githubusercontent.com/assets/79566/19908260/73bf4db0-a050-11e6-9290-706728ffc409.png)
